cdap-spark-core/src/main/java/co/cask/cdap/app/runtime/spark/SparkRuntimeContext.java

```diff
@@ -226,7 +226,7 @@ DynamicDatasetCache getDatasetCache() {
    * Returns the {@link WorkflowProgramInfo} if the spark program is running inside a workflow.
    */
   @Nullable
-  WorkflowProgramInfo getWorkflowProgramInfo() {
+  WorkflowProgramInfo getWorkflowInfo() {
```

Comment:
That will be the `co.cask.cdap.api.spark.SparkContext`. This internal `SparkRuntimeContext` doesn't implement that public interface, it just for carrying information/objects needed for the implementation.